src/meta.rst

```diff
@@ -215,12 +304,5 @@ the real package dependencies are only ``run`` requirements.
 
 Downloading extra sources and data files
 ----------------------------------------
-If you need additional source/data files for the build, download them using curl in the build script
-and verify the checksum using openssl. Add curl and openssl to the build requirements and then you
-can use curl to download and openssl to verify.
```

Comment:
Should we keep this text?

Reply:
Reason ?

Reply:
Was only thinking about the using `openssl` to verify checksum part. Agree the rest should be dropped.

Reply:
Each source can have a checksum, so there's no need to do that in the build script.

Reply:
Was thinking of [this]( https://github.com/conda-forge/staged-recipes/blob/06c6004f45e5000840262b420da5dceff01cb8a6/recipes/example/meta.yaml#L7-L10 ).

Anyways not a big deal. Am ok with dropping this.